.circleci/config.yml

```diff
@@ -0,0 +1,34 @@
+version: 2.0
+jobs:
+  build:
+    docker:
+      - image: grpc/python
+
+    steps:
+      - checkout
+      - run:
+          name: Install deps
+          command: pip install --no-cache-dir -r ./examples/python/requirements.txt
+
+      - run:
+          name: Start FullScriptSupport plugin
+          command: python ./examples/python/FullScriptSupport/ExtensionService_Script.py # :50051
```

Comment:
I think it may be a bit more clear if the port numbers are on the previous lines "name:", either as a comment or as actual part of the names.